package.json

```diff
@@ -22,7 +22,7 @@
     "node": ">=0.10"
   },
   "dependencies": {
-    "analyze-css": "0.10.x",
+    "analyze-css": "^0.10",
```

Comment:
Why not `^0.10.2` instead? This will install the latest version of `analyze-css`.